ocid.spec

```diff
@@ -0,0 +1,62 @@
+%define debug_package %{nil}
```

Comment:
This entire file should be in `contrib/rpm/redhat/...` or something like that. Not in the main directory. This actually also applies for `ocid.service` which should be in `contrib/systemd`.